src/bootstrap-table.js

```diff
@@ -976,6 +976,11 @@ class BootstrapTable {
       return value
     })
 
+    let paginationParts = opts.paginationParts
+    if (typeof paginationParts === 'string') {
+      paginationParts = paginationParts.replace(/\[|\]| /g, '').toLowerCase().split(',')
```

Comment:
Can save to `this.paginationParts` and then use it in other themes file.

Reply:
I think there is no need to use `toLowerCase` here, we can ask them to use the `keywords: ['pageInfo', 'pageSize', 'pageList']` according to the documentation.

Reply:
where is the advantage to save it to `this.paginationParts` instead to the variable ?